api/services/elastigroup/aws/paths/instance-lock.yaml

```diff
@@ -8,14 +8,6 @@ post:
   parameters:
     - $ref: "../../aws/parameters/instanceId.yaml"
     - $ref: "../../../../commons/parameters/accountId.yaml"
-    - in: "query"
-      name: "ttl"
```

Comment:
@vlad666spot why are you removing this query param?
the name should be fixed to 'ttlInMinutes' like in the old documentation -http://help.dev.spotinst.com/spotinst-api/elastigroup/amazon-web-services/lock-instances/

Reply:
![Screenshot 2020-11-04 at 15 43 48](https://user-images.githubusercontent.com/64230439/98118910-90f73a00-1eb4-11eb-9e01-985c87b400ea.png)
fixed